standards/licencing-software-or-code.md

```diff
@@ -0,0 +1,58 @@
+---
+category: Building software
+expires: 2018-01-31
+---
+# Licensing software or code
+
+Everything we produce or use, whether open or closed, should have an
+appropriate licence applied (as [indicated by the service
+manual][service manual licencing]). Without a licence, others (even
+within MOJ) have no way of knowing for sure if they can use it.
```

Comment:
... for sure if they are permitted by the originator to use the code or service, or under what terms or conditions.